app/app.js

```diff
@@ -1,18 +1,40 @@
 const express = require("express");
 const app = express();
-const { getCategory} = require("../controllers/app.controller")
+const { getCategory, getReviews, getReviewById} = require("../controllers/app.controller")
 
 app.use(express.json())
 
```

Comment:
I wonder why you've used an app.use(express.json()) before doing a post or a patch .... 📸